clang/lib/Headers/hlsl/hlsl_alias_intrinsics.h

```diff
@@ -9,6 +9,8 @@
 #ifndef _HLSL_HLSL_ALIAS_INTRINSICS_H_
 #define _HLSL_HLSL_ALIAS_INTRINSICS_H_
 
+#include "hlsl_compat_overloads.h"
```

Comment:
does it make more sense to add this to `hlsl.h` I don't think any of the alias intrinsics need this.

Reply:
Yeah I guess I was thinking of this as an extension of the hlsl_alias_intrinsics.h file, so I included it there, but I can include it in hlsl.h if that is more appropriate.